File: Modul6/C/PRAK602-2310817210020-Muhammad_Hasbiyan_Rusyadi.c
#include<stdio.h>
void zetsu(int ruangan, int jumlahzetsu[], int total[]) {
    for (int i = 0; i < ruangan; ++i) {
        total[i] = jumlahzetsu[i]*(i + 1);}}
int main() {
    int ruangan;
    scanf("%d", &ruangan);
    int jumlahzetsu[ruangan];
    int total[ruangan];
    for (int i = 0; i < ruangan; ++i) {
        scanf("%d", &jumlahzetsu[i]);}
    zetsu(ruangan, jumlahzetsu, total);
    for (int i = 0; i < ruangan; ++i) {
        printf("%d ", total[i]);}
    return 0;}

File: Modul6/Python/PRAK602-2310817210020-Muhammad_Hasbiyan_Rusyadi.py
def zetsu(ruangan, jumlahzetsu, total):
    for i in range(ruangan):
        total[i] = jumlahzetsu[i] * (i + 1)
def main():
    ruangan = int(input())
    jumlahzetsu = list(map(int, input().split()))
    total = [0] * ruangan
    zetsu(ruangan, jumlahzetsu, total)
    for i in range(ruangan):
        print(total[i], end=" ")
if __name__ == "__main__":
    main()
